ssl/statem/statem_lib.c

```diff
@@ -384,6 +384,7 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL *s, PACKET *pkt)
     }
 #ifndef OPENSSL_NO_GOST
     {
+        pktype = EVP_PKEY_id(pkey);
```

Comment:
It's OK for me to move them to the position of first usage.
Do we have any rule against the variable declaration: declare all variables in one location or declare them when we first use them?